CountryPicker/CountryPicker/PickerView/CountryPickerView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  CountryPickerView.swift
+//  CountryPicker
+//
+//  Created by Hardeep Singh on 05/12/19.
+//  Copyright © 2019 SuryaKant Sharma. All rights reserved.
+//
+
+import UIKit
+
+//public protocol ActionView {
+//    func showOnView(countryPicker: CountryPickerView)
+//}
+
+open class CountryPickerView: UIPickerView {
+    
+    private var selectedCountry: Country? {
+        didSet {
+            self.scrollToSelectedCountry()
+        }
+    }
+    
+    // ISO 3166-1 alpha-2 two-letter country codes.
+    private var countryCode: [String] = [String]() {
+        didSet {
+            self.updatePickList()
+        }
+    }
+    
+    private var allCountryList: [Country] = [Country]()
+    private var pickList: [Country] = [Country]() {
+        didSet {
+            self.reloadComponent(0)
+        }
+    }
+    
+    private func updatePickList() {
+        guard countryCode.count > 0 else {
```

Comment:
use isEmpty instead it bit more clear https://www.hackingwithswift.com/articles/181/why-using-isempty-is-faster-than-checking-count-0